bucket/splashtop-business-access

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://www.splashtop.com/",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://www.splashtop.com/terms/splashtop"
+    },
+    "description": "Splashtop is proprietary computer software for remote desktop operations.",
+    "version": "3.3.8.1",
+    "url": "https://d17kmd0va0f0mp.cloudfront.net/winclient/STB/Splashtop_Business_Win_PORTABLE_v3.3.8.1.exe",
+    "hash": "00d7e7da70744d57301e6fd855c048e8ec1a36df3efd76e7f0500929cf2aef23",
+    "pre_install": "New-Item $dir\\Splashtop_Business_Win_PORTABLE.exe | Out-Null",
+    "post_install": "Move-Item -Force $dir\\Splashtop_Business_Win_PORTABLE_v$version.exe $dir\\Splashtop_Business_Win_PORTABLE.exe",
+    "shortcuts": [
+        [
+            "Splashtop_Business_Win_PORTABLE.exe",
+            "Splashtop Business Access"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.splashtop.com/wp-content/themes/responsive/downloadx.php?product=stb&platform=windows-client-portable",
+        "jsonpath": "$.url",
+        "regex": "(?<URL>https:.*_v(?<version>\\d+\\.\\d+\\.\\d+\\.\\d+)\\.exe)"
+    },
+    "autoupdate": {
+        "url": "$matchURL"
+    }
+}
```

Comment:
```suggestion
{
    "version": "3.3.8.1",
    "description": "Fast, simple, secure remote computer access for individuals and teams",
    "homepage": "https://www.splashtop.com/business",
    "license": {
        "identifier": "Proprietary",
        "url": "https://www.splashtop.com/terms/splashtop"
    },
    "url": "https://d17kmd0va0f0mp.cloudfront.net/winclient/STB/Splashtop_Business_Win_PORTABLE_v3.3.8.1.exe#/splashtop.exe",
    "hash": "00d7e7da70744d57301e6fd855c048e8ec1a36df3efd76e7f0500929cf2aef23",
    "shortcuts": [
        [
            "splashtop.exe",
            "Splashtop Business Access"
        ]
    ],
    "checkver": {
        "url": "https://www.splashtop.com/wp-content/themes/responsive/downloadx.php?product=stb&platform=windows-client-portable",
        "jsonpath": "$.url",
        "regex": "([\\d.]+)\\.exe"
    },
    "autoupdate": {
        "url": "https://d17kmd0va0f0mp.cloudfront.net/winclient/STB/Splashtop_Business_Win_PORTABLE_v$version.exe#/splashtop.exe"
    }
}

```

Reply:
...

Reply:
oops. Totally missed those changes, just thought the whole file changed from the mv... Should be fixed now.